docker-compose.pinner.yml

```diff
@@ -16,8 +16,11 @@ services:
     logging: *default-logging
     env_file:
       - .env
+    volumes:
+      - ./docker/data/pinner/logs:/logs
     environment:
       - PINNER_LOG_LEVEL=${PINNER_LOG_LEVEL:-info}
+      - PINNER_LOG_FILE=${PINNER_LOG_FILE}
```

Comment:
if you're not assigning any defaults then this line is redundant as this service loads `.env` anyway

Reply:
you're right, I could have left it out. anyway, I'll just let it be, even if it's not strictly necessary.

Reply:
you're right. I thought about leaving it there but I'd rather start cleaning these files up. accounts has a whole bunch of these sitting around for no reason. we shouldn't do the same here. removed. :)